routes/refunds.js

```diff
@@ -3,13 +3,14 @@ var mongoose = require("mongoose");
 var Request = mongoose.model('Request');
 var express = require('express');
 var router = express.Router();
+var auth = require('../modules/auth');
 
-router.get('/refunds', function(req, res) {
+router.get('/refunds', auth.bounce, function(req, res) {
```

Comment:
Not a very big issue, but it's already being handled in the refunds branch.

AuthN and AuthZ for refunds is to be handled on that branch.
No problem though.